### PR TITLE
add work detail model

### PR DESCRIPTION
**Pull request by TSG-temp**
- [x] add work detail model
- [x] register work detail model
- [x] correct some typo

### CODE REVIEW

File: itez/beneficiary/models.py [hunk 1]
```diff
@@ -41,3 +41,28 @@ class District(models.Model):
...
+        null=False
+    )
+    insured = models.BooleanField(
+        "Company Insured",
+        default = False
+    )
+    work_address = models.CharField(
+        "Work Address",
+        max_length=300
+    )
```
Comment (sonlinux) [inline itez/beneficiary/models.py:69]: add the string method to the model

File: itez/beneficiary/models.py [hunk 2]
```diff
@@ -41,3 +41,28 @@ class District(models.Model):
...
+    Include Work Detail properties. 
+    """
+    salary = models.DecimalField( 
+        _("Monthly Salary"),
+        decimal_places=2,
+        max_digits=1000,
+        null=False
+    )
+    company = models.CharField(
+        "Company Name",
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: ensure you enclose field text in underscore `_("Company Name"),`

File: itez/beneficiary/models.py [hunk 3]
```diff
@@ -41,3 +41,28 @@ class District(models.Model):
...
+        max_length=200,
+        null=False
+    )
+    insured = models.BooleanField(
+        "Company Insured",
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: same here

File: itez/beneficiary/models.py [hunk 4]
```diff
@@ -41,3 +41,28 @@ class District(models.Model):
...
+        default = False
+    )
+    work_address = models.CharField(
+        "Work Address",
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: same here

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/models.py [hunk 5]
```diff
@@ -41,3 +41,31 @@ class District(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class WorkDetail(models.Model):
+    """
+    Include Work Detail properties. 
+    """
+    salary = models.DecimalField(
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: lets change this to `gross_pay`

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/models.py (hunk 5 above)
Comment (sonlinux) [reply itez/beneficiary/models.py]: that way we can have methods to compute `monthly_salary`

Comment (TSG-temp) [reply itez/beneficiary/models.py]: alright, working on it

File: itez/beneficiary/models.py [hunk 6]
```diff
@@ -41,3 +41,31 @@ class District(models.Model):
...
+    company = models.CharField(
+        _("Company Name"),
+        max_length=200,
+        null=False
+    )
+    insured = models.BooleanField(
+        _("Company Insured"),
+        default = False
+    )
+    work_address = models.CharField(
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: let this be a textfield 
`work_address=models.TextField(`
`                         _(Work Address), `
`                         null=True, `
`                         blank=True`
`)`

Comment (sonlinux) [reply itez/beneficiary/models.py]: then ensure to update migrations after

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/models.py [hunk 7]
```diff
@@ -41,3 +41,32 @@ class District(models.Model):
...
+    )
+    insured = models.BooleanField(
+        _("Company Insured"),
+        default = False
+    )
+    work_address = models.TextField(
+        _("Work Address"),
+        null=True,
+        blank=True
+        )
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: shift this closing bracket further to the left

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/models.py [hunk 8]
```diff
@@ -41,3 +41,32 @@ class District(models.Model):
...
+
+    def __str__(self):
+        return self.company
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: and leave one empty line  at  end of file every time

**Review by sonlinux — CHANGES_REQUESTED** [review]

**Review by sonlinux — APPROVED** [review]
LGTM

Thank you @chriford

**Pull request merged**